docs/howto/flask-async.rst

```diff
@@ -0,0 +1,53 @@
+======================================================
+How to write a Kubernetes charm for an Async Flask app
+======================================================
+
+In this how-to guide we will configure a 12-factor Flask
+application to use asynchronous Gunicorn workers to be
+able to serve to multiple users easily.
+
+Make the rock async
+===================
+
+Before packing the rock make sure to put the following in ``requirements.txt``
+file:
+
+.. literalinclude:: code/flask-async/requirements.txt
+
+Configure the async application
```

Comment:
@erinecon should we say that they should repack and deploy the charm with the new rock here?

Reply:
Yes, that's a good point. I think it could be a part of the `Make the rock async` section. Something like:

> Pack the rock using `rockcraft pack` and redeploy the charm with the new rock using `juju refresh`. 

If there's any concern about being too vague with the `juju refresh` command, then we could link to the Juju docs.